### PR TITLE
added a very basic readme

### PR DESCRIPTION
**Pull request by kailuowang**
Sorry @milessabin it is pretty preliminary. Didn't get as much as time to spend on it as I hoped. Let me know if there is anything you want to add.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+```
+More detailed explanations can also be found at this gist [Explaining Miles's Magic][explain] by @djspiewak and the readme of the [demo project][demo].
+
+
+
+## Usage
+
+For now you have to build the plugin yourself. Clone the project and run
+```bash
+sbt publishLocal
```
Comment (adelbertc) [inline README.md]: Maybe `+ publishLocal` because as of #2 it's on both Scala 2.10 and Scala 2.11 ;)

**Comment by mandubian** [comment]
👍 good start

**Comment by adelbertc** [comment]
👍

**Comment by milessabin** [comment]
Absolutely perfect! Many thanks :-)

**Pull request merged**